### PR TITLE
🐍 Battlesnake Quicksssssstart

### PR DESCRIPTION
**Pull request by zstix**
# Summary
A new quickstart that helps instrument your [Battlesnake](https://play.battlesnake.com)! ~At the moment, this includes one dashboard that shows server performance, but an upcoming PR will add a dashboard about Battlensake-specific stats (part 2).~ This includes both the server status and Battlesnake performance dashboard.

## Pre merge checklist
- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] Did you check that your alerts actually work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

#### Battlesnake Server Status
battlesnake-server-status

#### Battlesnake Performance
battlesnake-performance

### CODE REVIEW

File: install/battlesnake.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+id: battlesnake
+
+name: Custom Attributes
+
+title: APM: Report custom events and attributes
```
Comment (aswanson-nr) [inline install/battlesnake.yml:5]: The `APM:` here is causing yaml parsing errors. Moving it to a piped field should alleviate the issue
```suggestion
title: |
  APM: Report custom events and attributes
```

Comment (aswanson-nr) [reply install/battlesnake.yml:5]: Hmm, that syntax highlighting has me worried

**Review by aswanson-nr — APPROVED** [review]
Looks good to me!

File: install/battlesnake/install.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+id: battlesnake
+
+name: Custom Attributes
+
+title: |
+  APM: Report custom events and attributes
+
```
Comment (jpvajda) [inline install/battlesnake/install.yml:7]: Please add a summary as, so it's not blank on the catalog view.

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: install/battlesnake/install.yml (hunk 2 above)
Comment (aswanson-nr) [reply install/battlesnake/install.yml:7]: @jpvajda This is the install plan! So no summary needed here.

File: install/battlesnake/install.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+id: battlesnake
+
+name: Custom Attributes
+
+title: |
```
[the remark this answers is not in the document]
Comment (zstix) [reply install/battlesnake/install.yml:5]: This was copied from an existing install plan. Not sure if this is required or not?

[the remark this answers is not in the document]
Comment (aswanson-nr) [reply install/battlesnake/install.yml:5]: FYI: The title is used and transformed to `heading` when passed to the nr1Catalog API.

[the remark this answers is not in the document]
Comment (jpvajda) [reply install/battlesnake/install.yml:5]: Ah.. That makes sense as this is what I saw in the docs, though it's a bit confusing so I guess I can capture this in our contributing guide, that title is now heading.

Should we consider changing all install plans that have title > heading?

heading | String! | Used as a heading for the install plan step
-- | -- | --

**Review by jpvajda — APPROVED** [review]

File: install/battlesnake/install.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jpvajda) [reply install/battlesnake/install.yml:5]: I'll make a ticket for that.

**Pull request merged**